1.4/python/alpine/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-alpine as base
+
+FROM base as builder
+
+RUN apk add --update --no-cache \
+    gcc \
+    linux-headers \
+    make \
+    musl-dev \
+    python-dev \
+    g++
+
+ENV GRPC_PYTHON_VERSION 1.4.0
```

Comment:
shouldn't this be  `1.14.0` instead?

Reply:
Not for this particular image. I was proposing adding new tags for already existing "types" in the repository. 

If the team is okay with it, I will open more PRs for additional versions of gRPC